### ISSUE TITLE
FAQ/addition to documentation on Download file generation

### ISSUE THREAD

**Issue by IgorRodchenkov** [PathwayCommons/cpath1#34]
_From @GoogleCodeExporter on March 16, 2015 20:15_

```
Peter Carbonetto (pcarbo@uchicago.edu)

Add an FAQ and a paragraph in the readme describing the generation of the Gene 
set download files.
```

Original issue reported on code.google.com by `Nad.An...@gmail.com` on 14 Jan 2011 at 8:10

_Copied from original issue: PathwayCommons/cpath2#39_

**Comment by IgorRodchenkov** [comment]
_From @GoogleCodeExporter on March 16, 2015 20:15_

Original comment by `rod...@gmail.com` on 4 Jan 2013 at 4:31
- Added labels: **Type-Task**
- Removed labels: **Type-Enhancement**

**Comment by IgorRodchenkov** [comment]
_From @GoogleCodeExporter on March 16, 2015 20:15_

Original comment by `rod...@gmail.com` on 4 Jan 2013 at 4:32
- Added labels: **Product-cPath**, **Product-cPath2**

**Comment by IgorRodchenkov** [comment]
_From @GoogleCodeExporter on March 16, 2015 20:15_

Original comment by `rod...@gmail.com` on 14 Feb 2013 at 5:23
- Added labels: **Milestone-M2**

**Comment by IgorRodchenkov** [comment]
_From @GoogleCodeExporter on March 16, 2015 20:15_

Original comment by `rod...@gmail.com` on 14 Feb 2013 at 5:24
- Added labels: **Sprint-Sprint-9**

**Comment by IgorRodchenkov** [comment]
_From @GoogleCodeExporter on March 16, 2015 20:15_

```
This issue was closed by revision a8cd4164661d.
```

Original comment by `rod...@gmail.com` on 27 May 2013 at 7:28
- Changed state: **Fixed**
